### ISSUE TITLE
Allow skipping of install step

### ISSUE THREAD

**Issue by Limess** [cypress-io/circleci-orb#59]
Our general circle workflow is to do an install using `npm ci`, and then to fan out to the jobs we need to run in parallel, attaching the initial workspace to each job.

This is not possible when using: `cypress/run` and results in duplicate install logic/time, using up an additional container.

Additionally, we sometimes still use other install steps. One such example is `bower install` (or front-end component library http://origami.ft.com/ still uses Bower) - adding a `./node_modules.bin/bower install` `build` parameter increases duplication, skips the existing circle bower cache, and requires an additional `--allow-root` flag due to the container used.

It'd be a nice to have to be able to skip the `cypress/install` step by passing a flag, but still attach the workspace at the correct location.

With appropriate documentation this may remove the need for modification for https://github.com/cypress-io/circleci-orb/issues/28

**Comment by bahmutov** [comment]
yeah, the problem with custom installs - do we still need to cache folders, and are they the right folders to cache? I would be open to a pull request that shows how to do this. Maybe at least disable / enable install step via a parameter (which should also disable caching I believe, if possible)

**Comment by clarmso** [comment]
I was looking for this feature, too. I'd like to build once and then run the tests (possibly in parallel) using different versions of Chrome.

My workaround is to set the `parallel` flag to `true` in the `cypress/run` step. The tests are not necessarily run in parallel, but this flag attaches the existing workspace and avoids the `cypress/run` job from running build again.
https://github.com/cypress-io/circleci-orb/blob/f33b1d5c343d3abac6876ad810d575bacfb2dc04/src/orb.yml#L342

Note that there's a cost to attach an existing workspace. For very small projects, you may not save very much time by doing so.

**Comment by bahmutov** [comment]
In both cases I suggest writing your own install job (which can reuse `install` command from this orb), building the app there and then making the `cypress/run` job depend on your custom `my-install` job. Set `parallel: true` flag to avoid `cypress/run` invoking its install command.

See https://github.com/cypress-io/circleci-orb#parallel for example
